deploy/helm/magda/charts/cloud-sql-proxy/templates/service.yaml

```diff
@@ -0,0 +1,17 @@
+{{- if and .Values.global.exposeNodePorts .Values.global.useCloudSql }}
+apiVersion: v1
+kind: Service
+metadata:
+  name: cloud-sql-proxy
+spec:
+  ports:
+  - name: postgres
+    port: 5432
+    targetPort: 5432
+{{- if .Values.global.exposeNodePorts }}
```

Comment:
But this whole servie is only defined if `exposeNodePorts` is true. Is this check redundant, or is the check at the top an error?

Reply:
Oops, yeah don't need that one.